smoke_tests/delegations_test.js

```diff
@@ -0,0 +1,226 @@
+// k6 smoke test of api delegations actions
```

Comment:
let's discuss on slack how to make this reusable between the api and auth CIs

Reply:
Is this something for this PR, or future work?

Reply:
I'm thinking moving the updated scripts back to liquidvotingio/deployment and then using them from there after [checking out the repo](https://github.com/actions/checkout/issues/95) there might be low-enough hanging fruit?

So submitting a separate PR on liquidvotingio/deployment with the updated smoke tests, and then updating this PR here to point it.

Reply:
so...
1. Move new versions of k6 test out of api and into deployment
2. Move the smoke test actions to a deployment repo. action (that can also be called manually)?
3. add an action to api.repo `on_push.yml` that calls the action in the deployment repo. (to run the smoke test action)?

Reply:
yes, exactly :)